### DISCUSSION TITLE
Feature Request - annotations (UI?)

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#8027] [Ideas]
**Is your feature request related to a problem? Please describe.**

@Jegelewicz @AJLinn - from AWG meeting - how can annotations be better?

**Comment by AJLinn** [comment]
Wanna chat about it at the next Arctos Lunch? Mondays are usually pretty open for my schedule. 
Mostly I think we need a better way to understand what the annotation is addressing and what i, as the manage collection person, needs to do about it. Then, once I've done it, figure out how to hide or archive the annotation. I just find it utterly unusable right now.

**Comment by Jegelewicz** [comment]
I'll add it to the agenda the week before!

**Comment by dustymc** [comment]
>Arctos Lunch

My summer schedule is still a bit waffly and hectic, but I _think_ I'll be around and that's now on my calendar.

**Comment by mkoo** [comment]
When is Arctos Lunch? Not seeing it on Arctos calendar!

OK to use this Label? or do we need another for the Arctos lunches? if not, this helps me find it!

**Comment by dustymc** [comment]
Monday, July 10⋅11:30am – 12:30pm is what I'm looking at! (Surely there's some way to get a link to a calendar event!?)

**Comment by mkoo** [comment]
thanks-- I see you're living in the future! I didnt view the calendar for July... !

**Comment by Jegelewicz** [comment]
Functional Requirements started at https://docs.google.com/document/d/1hBkxxEkbVvfFpQHPL2BaOy-3dEQA7uUuXRFWIUvLX_8/edit?usp=sharing
